hrw.go

```diff
@@ -28,7 +29,9 @@ type (
 	}
 )
 
-func weight(x uint64, y uint64) uint64 {
+const normalizedMaxWeight = 1.0
```

Comment:
It is better to make it public, so every normalizing function can use it.
What do you think?